python/animl/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM socrata/python
+
+MAINTAINER Nick Kypreos <nick.kypreos@socrata.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install python-numpy python-scipy
+
+# pip installations
+RUN pip install geopy==1.5.0
```

Comment:
I'll buy that, but not for the slow part because of caching and automation. We don't need to create a layer for each install, though, 'cause it may bloat pretty hard.